docs/datasets/danish.md

```diff
@@ -354,6 +354,41 @@ $ scandeval --model <model-id> --dataset scandiqa-da
 ```
 
 
+### Belebele
+
+This dataset was published in [this paper](https://aclanthology.org/2024.acl-long.44/) and is a large-scale multilingual reading comprehension dataset covering 122 languages. The questions are generated from Wikipedia articles and are designed to test various aspects of reading comprehension, including factual understanding, inference, and numerical reasoning.
+
+The dataset provides training, validation, and test splits with human-verified question-answer pairs. The questions are generated to be answerable from the given context and cover diverse topics from Wikipedia articles.
```

Comment:
We need to specify both the original splits as well as the splits that we use - this can be found in the `src/scripts/create_belebele.py` script.